Fix integration test to work on py35

Dictionaries are *not ordered* prior to py36:
```
FAIL: test_maze (tests.test_integration.IntegrationTests) [Verify if output contains '{'0,1w':' 
]
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\circleci\repo\tests\test_integration.py", line 32, in test_maze
    self.assertIn("{'0,1w':".upper(), output)
AssertionError: "{'0,1W':" not found in 'B"#|#######\\R\\N#._._._.#\\R\\N#  #   |#\\R\\N#. . . .#\\R\\N#      |#\\R\\N#. .#. ._\\R\\N#########\\R\\N\\R\\N\\R\\N{\'AUX7\': 0, \'0,3W\': 1, \'1,1W\': 0, \'1,3N\': 1, \'0,1W\': 1, \'2,0N\': 0, \'2,1W\': 0, \'AUX6\': 1, \'2,3W\': 0, \'1,0N\': 0, \'AUX8\': 0, \'AUX9\': 1, \'1,2N\': 0, \'2,3N\': 1, \'2,1N\': 0, \'AUX0\': 0, \'1,3W\': 0, \'1,2W\': 0, \'AUX1\': 0, \'AUX5\': 0, \'AUX4\': 0, \'AUX3\': 0, \'AUX2\': 0, \'2,2N\': 0, \'0,2W\': 1}\\R\\N"'
```

--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -28,7 +28,7 @@ class IntegrationTests(unittest.TestCase):
         if os.getenv('DEBUG_OUTPUT'):
             print("Example output \n" + output)
 
-        with self.subTest(msg="Verify if output contains '0,1w':' \n"):
+        with self.subTest(msg="Verify if output contains \"'0,1w':\" \n"):
             self.assertIn("'0,1w':".upper(), output)
         with self.subTest(msg="Verify if error string contains in output \n"):
             self.assertNotIn("ERROR", output)

--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -28,8 +28,8 @@ class IntegrationTests(unittest.TestCase):
         if os.getenv('DEBUG_OUTPUT'):
             print("Example output \n" + output)
 
-        with self.subTest(msg="Verify if output contains '{'0,1w':' \n"):
-            self.assertIn("{'0,1w':".upper(), output)
+        with self.subTest(msg="Verify if output contains '0,1w':' \n"):
+            self.assertIn("'0,1w':".upper(), output)
         with self.subTest(msg="Verify if error string contains in output \n"):
             self.assertNotIn("ERROR", output)
         with self.subTest(msg="Verify if warning string contains in output \n"):

